[Ready] More useful tips. (#7957)
Citadel-Station-13/Citadel-Station-13-6th-Port: strings/tips.txt

Maintenance is full of equipment that is randomized every round. Look around and see if anything is worth using.
Some roles cannot be antagonists by default, but antag selection is decided first. For instance, you can set Security Officer to High without affecting your chances of becoming an antag -- the game will just select a different role.
There are many places around the station to hide contraband. A few for starters: linen boxes, toilet cisterns, body bags. Experiment to find more!

> On all maps, you can use a machine in the vault to deposit space cash for cargo points or rob cargo, but if you print out money it blares a alarm to the crew!

Reviewer: ```suggestion
On all maps, you can use a machine in the vault to deposit space cash for cargo points. Otherwise, use it to steal the station's cash and get out before the alarm goes off.
```

As the Captain, always take the nuclear disk and pinpointer with you every shift. It's a good idea to give one of these to another head you can trust with keeping it safe, such as the Head of Security.
As the Captain, you have absolute access and control over the station, but this does not mean that being a horrible person won't result in mutiny and a ban.
As the Captain, you have a fancy pen that can be used as a holdout dagger or even as a scalpel in surgery!
As the Captain, you can purchase a new emergency shuttle using a communications console. Some require credits, while others give you credits in exchange. Keep in mind that purchasing dangerous shuttles will incur the ire of your crew.
As the Chief Medical Officer, your hypospray is like a refillable instant injection syringe that can hold 30 units as opposed to the standard 15.
As the Chief Medical Officer, coordinate and communicate with your doctors, chemists, and geneticists during a nuclear emergency, blob infestation, or some other crisis to keep people alive and fighting.

> As a Medical Doctor, you can print better surgical tools with the help of RnD, these tools are faster and some can be used as lethal weapons!

Reviewer: ```suggestion
As a Medical Doctor, pester Research for improved surgical tools. They work faster, don't cost much and are typically more deadly.
```

> As a Medical Doctor, your belt can hold a full set of surgical tools, and using a spray of stailizer before each step will make you more likely to successfully complete the step!

Reviewer: ```suggestion
As a Medical Doctor, your belt can hold a full set of surgical tools. Using sterilizine before each attempt during surgery will reduce your failure chance on tricky steps or when using less-than-optimal equipment.
```

As a Geneticist, you can eject someone from cloning early by clicking on the cloner pod with your ID. Note that they will suffer more genetic damage and may lose vital organs from this.
As a Geneticist, becoming a hulk makes you capable of dealing high melee damage, stunlocking people, and punching through walls. However, you can't fire guns, will lose your hulk status if you take too much damage, and are not considered a human by the AI while you are a hulk.
As the Virologist, your viruses can range from healing powers so great that you can heal out of critical status, or diseases so dangerous they can kill the entire crew with airborne spontaneous combustion. Experiment!
As the Virologist, you only require small amounts of vaccine to heal a sick patient. Work with the Chemist to distribute your cures more efficiently.
As the Research Director, you can take AIs out of their cores by loading them into an intelliCard, and then from there into an AI system integrity restorer computer to revive and/or repair them.
As the Research Director, you can lock down cyborgs instead of blowing them up. Then you can have their laws reset or if that doesn't work, safely dismantled.

> As the Research Director, you can upgrade your modular console with better computer parts to speed up its fuctions. Like the AI system integrity restorer.

Reviewer: ```suggestion
As the Research Director, you can upgrade your modular console with better computer parts to speed up its functions. This can be useful when using the AI system integrity restorer.
```

As a Scientist, you can use the mutation toxin obtained from green slimes to turn yourself into a jelly mutant. Each subspecies has unique features - for example telepathic powers, duplicating bodies or integrating slime extracts!
As a Scientist, you can maximize the number of uses you get out of a slime by feeding it slime steroid, created from purple slimes, while alive. You can then apply extract enhancer, created from cerulean slimes, on each extract.
As a Scientist, you can disable anomalies by scanning them with an analyzer, then send a signal on the frequency it gives you with a remote signalling device. This will leave behind an anomaly core, which can be used to construct a Phazon mech, or be used in the destructive analyzer for a 10,000 point bonus!
As a Scientist, researchable stock parts can seriously improve the efficiency and speed of machines around the station. In some cases, it can even unlock new functions.
As a Scientist, you can generate research points by letting the tachyon-doppler array record increasingly large explosions.
As a Scientist, getting drunk just enough will speed up research. Skol!

> As a Scientist, you can get points by placing slime cores into the deconstructive analyzer! This even works with crossbreed slime cores. 

Reviewer: ```suggestion
As a Scientist, you can get points by placing slime cores into the deconstructive analyzer! This even works with crossbred slime cores. 
```

As a Roboticist, keep an ear out for anomaly announcements. If you get your hands on an anomaly core, you can build a Phazon mech!
As a Roboticist, you can repair your cyborgs with a welding tool. If they have taken burn damage from lasers, you can remove their battery, expose the wiring with a screwdriver and replace their wires with a cable coil.
As a Roboticist, you can reset a cyborg's module by cutting and mending the reset wire with a wire cutter.
As a Roboticist, you can greatly help out Shaft Miners by building a Firefighter APLU equipped with a hydraulic clamp and plasma cutter. The mech is ash storm proof and can even walk across lava!
As a Roboticist, you can augment people with cyborg limbs. Augmented limbs can easily be repaired with cables and welders.

> As a Roboticist, you can use your printer that is linked to the ore silo to teleport mats into your work place!

Reviewer: what

Author: Tis true

Reviewer: It's written like word salad. What printer?
```suggestion
As a Roboticist, you can use your (circuit?/techfab?/copying machine?) printer that is linked to the ore silo to teleport mats into your work place!
```

As a Cyborg, you are extremely vulnerable to EMPs as EMPs both stun you and damage you. The ion rifle in the armory or a traitor with an EMP kit can kill you in seconds.
As a Service Cyborg, your spray can knocks people down. However, it is blocked by gas masks.
As an Engineering Cyborg, you can attach air alarm/fire alarm/APC frames to walls by placing them on the floor and using a screwdriver on them.

> As a Medical Cyborg, you can fully perform surgery and even augment people. Better yet you can't fail to perform surgery, even in the worst of conditions!

Reviewer: ```suggestion
As a Medical Cyborg, you can fully perform surgery and even augment people. Best of all, they have a 0% failure chance.
```

As an Engineer, don't underestimate the humble P.A.C.M.A.N. generators. With upgraded parts, a couple units working in tandem are sufficient to take over for an exploded engine or shattered solars.
As an Engineer, your departmental protolathe and circuit printer can manufacture the necessary circuit boards and components to build just about anything. Make extra medical machinery everywhere! Build a gibber for security! Set up an array of emitters pointing down the hall! The possibilities are endless!
As an Engineer, you can pry open secure storage by disabling the engine room APC's main breaker. This is obviously a bad idea if the engine is running.

> As an Engineer, make sure to yell at cargo for some meteor shields to set up around the SME and armory!

Reviewer: ```suggestion
Don't forget that Cargo has access to a meteor defense satellite that can be ordered BEFORE meteors hit the station. Any idle Engineers should have this on their to-do list.
```

As an Atmospheric Technician, your ATMOS holofan projector blocks gases while allowing objects to pass through. With it, you can quickly contain gas spills, fires and hull breaches. Or, use it to seal a plasmaman cloning room.
As an Atmospheric Technician, burning a plasma/oxygen mix inside the incinerator will not only produce power, but also gases such as tritium and water vapor.
As an Atmospheric Technician, you can change the layer of a pipe by clicking with it on a wrenched pipe or other atmos component of the desired layer.

> As an Atmospheric Technician, you can take a few cans worth of N2/N2O and cool it down at local freezers. This is a good idea when dealing with a SME meltdown. 

Reviewer: ```suggestion
As an Atmospheric Technician, you can take a few cans worth of N2/N2O and cool it down at local freezers. This is a good idea when dealing with (or preparing for) a supermatter meltdown. 
```

As the Head of Security, you are expected to coordinate your security force to handle any threat that comes to the station. Sometimes it means making use of the armory to handle a blob, sometimes it means being ruthless during a revolution or cult.
As the Head of Security, you can call for executions or forced cyborgization, but may require the Captain's approval.
As the Head of Security, don't let the power go to your head. You may have high access, great equipment, and a miniature army at your side, but being a terrible person without a good reason is grounds for banning.
As the Warden, your duty is to be the watchdog of the brig and handler of prisoners when little is happening, and to hand out equipment and weapons to the security officers when a crisis strikes.
As the Warden, keep a close eye on the armory at all times, as it is a favored strike point of nuclear operatives and cocky traitors.
As the Warden, if a prisoner's crimes are heinous enough you can put them in permabrig or the gulag. Make sure to check on them once in a while!

> As the Warden, never underestimate the power of tech slugs! Scattershot fires a cone of weaker lasers, Ion slugs fires EMPs that only effect the tiles they hit, and Pulse slugs fire a singular laser that can one-hit almost any wall!

Reviewer: ```suggestion
Never underestimate the power of tech slugs! Scattershot fires a cone of weaker lasers, Ion slugs fires EMPs that only effect the tiles they hit, and Pulse slugs fire a singular laser that's almost as strong as a pulse rifle.
```

Reviewer: This isn't a warden-specific tip, hence why I changed it.

> As the Warden, you can use a surgery saw on riot shotguns to shorten the barrel, making them able to fit in bags!

Reviewer: ```suggestion
As the Warden, you can use a surgical saw on riot shotguns to shorten the barrel, making them able to fit in your backpack.
```

As the Warden, you can implant criminals you suspect might re-offend with devices that will track their location and allow you to remotely inject them with disabling chemicals. 
As the Warden, you can use handcuffs on orange prisoner shoes to turn them into cuffed shoes, forcing prisoners to walk and potentially thwarting an escape.
As the Warden, tracker implants can be used on sec officers. Doing this will let you track their corpse even without suits, though the implant will biodegrade after 5 minutes.

> As the Warden, cryostasis shotgun dart hold 10u of mix of chems, like a 10u of meth bomb!

Reviewer: Try again, please.

Author: Sure

As the Warden, chemical implants can be loaded with a cocktail of healing or combat chems, perfect for the Hos or other sec officers to use. Be sure to keep a eye on them though, it will not auto inject! EMPs or starvation mite lead to the chemical implant to go off as well.

> As the Warden, tracker implants can be used on sec officers. Doing this will let you be able to message them when telecoms are out, or when you suspect coms are breached! This is also good against rogue AIs as the prisoner tracker doesn't leave logs or alarms for the AI!

Reviewer: ```suggestion
As the Warden, tracker implants can be used on sec officers. Doing this will let you be able to message them when telecoms are out, or when you suspect coms are compromised. This is also good against rogue AIs as the prisoner tracker doesn't leave logs or alarms for the AI.
```

As a Security Officer, remember that correlation does not equal causation. Someone may have just been at the wrong place at the wrong time!
As a Security Officer, remember that your belt can hold more then one stun baton. 

> As a Security Officer, remember harm battoning someone in the head can deconvert them form a being a rev! This sadly doesn't work against the cult, nor does this protect them form getting reconverted.

Reviewer: ```suggestion
As a Security Officer, remember harm battoning someone in the head can deconvert them form a being a rev! This sadly doesn't work against the cult, nor does this protect them from getting reconverted.
```

As a Cook, you can rename your custom made food with a pen.
As a Cook, any food you make will be much healthier than the junk food found in vendors. Having the crew routinely eating from you will provide minor buffs.
As a Cook, being in the kitchen will make you remember the basics of Close Quarters Cooking. It is highly effective at removing Assistants from your workplace.

> As a Cook, you can vend out trays that fit on your blet slot! These trays hold a plethora of different objects!

Reviewer: ```suggestion
As a Cook, your Kitchenmate can vend out trays that fit on your belt slot. These trays pick up 7 food items at a time and are a quick way to transport large meals.
```

> As a Cook, the advanced roasting stick is used to cook food at a distance, and can be used on SME, singularity, and other objects that cook food normally!

Reviewer: ```suggestion
As a Cook, the advanced roasting stick is used to cook food at a distance, and can be used on SME, singularity, and other objects that cook food normally.
```

As the Bartender, the drinks you start with only give you the basics. If you want more advanced mixtures, look into working with chemistry, hydroponics, or even mining for things to grind up and throw in!
As the Bartender, you can use a circular saw on your shotgun to make it easier to store.
As a Janitor, if someone steals your janicart, you can instead use your space cleaner spray, grenades, water sprayer, exact bloody revenge or order another from Cargo.

> As a Janitor, your trash bag is a robust storge belt item! It can not only hold a full set of tools, but a ton of different medical or small weapons!

Reviewer: ```suggestion
As a Janitor, the trash bag can be used to hold more than trash. Tools, medical equipment, smuggled nuclear disks... You name it!
```

As a Janitor, mousetraps can be used to create bombs or booby-trap containers.

> As the Curator, you are not completely defenseless. Your whip always disarms people, your laser pointer can blind humans and cyborgs, and you can hide items in wirecut books.

Reviewer: ```suggestion
Beware the Curator, for they are not completely defenseless. The curator's whip always disarms people, their laser pointer can blind humans and cyborgs, and can hide items in wirecut books.
```

As the Curator, be sure to keep the shelves stocked and the library clean for crew.
As a Cargo Technician, you can hack MULEbots to make them faster, run over people in their way, and even let you ride them!
As a Cargo Technician, you can order contraband items from the supply shuttle console by de-constructing it and using a multitool on the circuit board, the re-assembling it.
As a Cargo Technician, you can earn more cargo points by shipping back crates from maintenance, liquid containers, plasma sheets, rare seeds from hydroponics, and more!

> As a Cargo Technician, you get 400 points per packet! Stamp the manifest and sending back the crate will give you 200 points for the paperwork and 200 points for the crate!

Reviewer: This is totally messed up. Zero context to hint at just what the frick you're talking about.

Author: It cant be any more clear then that

Reviewer: WHAT PACKETS? WHAT STAMPS? WHERE DO I GET PACKETS?? REEEEEEEEEEEEEEE

> As a Cargo Technician, you can now buy paper work! Go into the crafting menu to turn pens and undone paper work into filed paper work to get 50 points per!

Reviewer: ```suggestion
As a Cargo Technician, paperwork is an alternative option to shipping off plasma sheets and other goods. Order Paperwork crates and go into the crafting menu to turn pens and undone paper work into completed grant paper work to get 50 points per sheet!
```

As the Malfunctioning AI, you can shunt to an APC if the situation gets bad. This disables your doomsday device if it is active.
As the Malfunctioning AI, you should either order your cyborgs to dismantle the robotics console or blow it up yourself in order to protect them.
As the Malfunctioning AI, look into flooding the station with plasma fires to kill off large portions of the crew, letting you pick off the remaining few with space suits who escaped.

> Xeno invasion? Bug science to give you some tech shells so you can craft Pulse slugs and Scattershot!

Reviewer: ```suggestion
Xenomorphs? Science can craft deadly tech shells like pulse slugs and frag shot that are highly effective against any alien threat.
```

When fighting aliens, it can be a good idea to turn off the gravity due to the alien's lack of zero-gravity control.

> When fighting aliens, grab a shield! You can block their pounces, though be careful of the neurotoxin!

Reviewer: ```suggestion
When fighting xenomorph aliens, consider a shield. Shields can block their pounces and be worn on the back, but beware of neurotoxin.
```

As a Blobbernaut, you can communicate with overminds and other Blobbernauts via :b.
As a Blobbernaut, your HUD shows your health and the core health of the overmind that created you.
As a Revolutionary, you cannot convert a head of staff or someone who has a mindshield implant, such as a security officer or those they implant. Implants can however be surgically removed, and do not carry over with cloning. Take control of medbay to keep control of conversions!

> As a Revolutionary, you cannot convert someone with brain damage, making a un-upgraded cloner a getto deconverter.

Reviewer: ```suggestion
During a revolution, you cannot convert someone with brain damage, making an un-upgraded cloner a makeshift deconversion device.
```

As a Revenant, the illness inflicted on humans by Blight can be easily cured by lying down or with holy water, making it best used on targets that have no time to lie down, such as humans in combat.
As a Swarmer, you can deconstruct more things than you think. Try deconstructing light switches, buttons, air alarms and more. Experiment!
As a Swarmer, you can teleport fellow swarmers away if you think they are in danger.

> As a Swarmer, you are weak to EMP and lasers; and if you die, you drop a blue space crystal.

Reviewer: ```suggestion
As a Swarmer, you are weak to EMP and lasers. Upon death, you will drop a bluespace crystal.
```

> As a Swarmer, you can eat almost all guns and weapons be sure to disarm the crew to keep your numbers high!

Reviewer: ```suggestion
As a Swarmer, use your ability to consume guns and other weapons to disarm the crew to keep your numbers high!
```

As a Morph, you can talk while disguised, but your words have a chance of being slurred, giving you away!
As a Drone, you can ping other drones to alert them of areas in the station in need of repair.

> As a Drone, you can heal yourself by using a screwdriver on yourself and standing still!

Reviewer: ```suggestion
As a Drone, you can repair yourself when using a screwdriver and standing still!
```

As a Ghost, you can see the inside of a container on the ground by clicking on it.
As a Ghost, you can double click on just about anything to follow it. Or just warp around!
As a Devil, you gain power for every three souls you control, however you also become more obvious.
As a Devil, as long as you control at least one other soul, you will automatically resurrect, as long as a banishment ritual is not performed.
At which time a Devil's nameth is spake on the tongue of man, the Devil may appeareth. 
You can swap floor tiles by holding a crowbar in one hand and a stack of tiles in the other.
When hacking doors, cutting and mending the "test light wire" will restore power to the door.
When crafting most items, you can either manually combine parts or use the crafting menu.
Suit storage units not only remove blood and dirt from clothing, but also radiation!
Remote devices will work when used through cameras. For example: Bluespace RPEDs and door remotes.

> Laser pointers can be upgraded by replacing its micro laser with a better one form RnD! Just use a screwdriver on it to remove the old laser. Upgrading the laser pointer gives you better odds of stunning a borg, and even blinding people with sunglasses!

Reviewer: ```suggestion
Laser pointers can be upgraded by replacing its micro laser with a better one from RnD! Use a screwdriver on it to remove the old laser. Upgrading the laser pointer gives you better odds of stunning a cyborg, and even blinding people with sunglasses.
```
